src/templates/main/home.html

```diff
@@ -1,5 +1,11 @@
 {% extends 'main/base.html' %}
+
+{% load vis_echarts_components %}
+
 {% block title %}Home{% endblock %}
+
 {% block content %}
-<h1>Home</h1>
+
```

Comment:
It would be nice to add Some text in this page So that we don't have just the visualization

Reply:
> It would be nice to add Some text in this page So that we don't have just the visualization
Sure!